### PR TITLE
Hyundai: add FW for 2019 AZERA_HEV_6TH_GEN

### PR DESCRIPTION
**Pull request by chanhojung**

### CODE REVIEW

**Comment by sshane** [comment]
What is your dongle id?

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -555,18 +558,24 @@ def match_fw_to_car_fuzzy(live_fw_versions) -> Set[str]:
...
+      b'\xf1\x00IG  MDPS C 1.00 1.01 56310M9350\x00 4IH8C101',
     ],
     (Ecu.fwdRadar, 0x7d0, None): [
       b'\xf1\x00IGhe SCC FHCUP      1.00 1.00 99110-M9100         ',
+      b'\xf1\x00IGhe SCC FHCUP      1.00 1.01 99110-M9000         ',
     ],
     (Ecu.transmission, 0x7e1, None): [
       b'\xf1\x006T7N0_C2\x00\x006T7VA051\x00\x00TIGSH24KA1\xc7\x85\xe2`',
+      b'\xf1\x006T7N0_C2\x00\x006T7Q2051\x00\x00TIG2H24KA2\x12@\x11\xb7',
+      b'\xf1\x816T7Q2051\x00\x00\xf1\x006T7N0_C2\x00\x006T7Q2051\x00\x00TIG2H24KA2\x12@\x11\xb7',
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:574]:
```suggestion
```

**Comment by sshane** [comment]
Can you try this branch, not your custom branch? Also, does this work without your custom changes to the LKAS message for the HUD alerts that you recently reverted?

**Comment by sshane** [comment]
You don't need to install stock (master or release), you can install the branch you opened a PR with (patch-3) on your repo.

**Comment by sshane** [comment]
Thanks! No errors on your dash?

**Comment by Lotx-001** [comment]
I know it's your car, but the specifications are the same as the Kia Cadenza 😊 MDPS-C, LKAS, ASCC

**Pull request merged**